### DISCUSSION TITLE
getting sites / sitegroups

### DISCUSSION THREAD

**Discussion by rauwebieten** [craftcms/element-api#149] [Q&A] [answered]
### Description

Is there any way to query all sites and/or sitegroups ?

**Comment by brandonkelly** [comment, marked as answer]
You would need to define a custom [Fractal resource](https://fractal.thephpleague.com/resources/) class, and point your endpoint to it via the [class](https://github.com/craftcms/element-api#class) setting. Note that if you do that, most other endpoint config settings would no longer apply.
